pyblish_lite/control.py

```diff
@@ -146,6 +146,7 @@ def load_plugins(self):
         self.was_discovered.emit()
 
     def on_collected(self):
+        self.current_error = None
```

Comment:
Not sure about this one; can't we just have the test determine when to stop? Some people may have implemented a test that stop at arbitrary orders, including within the collection stage. If we hard-code this particular order then those tests might not work.

The test is already used to determine when to stop [later in the code](https://github.com/pypeclub/pyblish-lite/blob/104ea10a7ed45279d29ca8a42e2d0a1e6bf155f3/pyblish_lite/control.py#L221), and if `current_error` is also involved in when to stop then we've got two tests; one editable, and the other not.

Reply:
Well, then I don't know how to be able to continue after collector's failure. I think you'll need to choose one...